src/main/java/anab/Person/ManagePerson.java

```diff
@@ -13,22 +18,62 @@ public static void main(String[] args) {
         System.out.println("Last Name:\t" + personPerson.getLastName());
         System.out.println();
         String name = "Buruiana";
-        printeazaText("Ana " + name+".");
+        printeazaText("Ana " + name);
         System.out.println();
-
         personPerson.setGender("Male");
         System.out.println("Gender type: " + personPerson.getGender());
         System.out.println();
-
         String newEmailAddress = "myEmail@google.com";
         personPerson.setEmailAddress(newEmailAddress);
         System.out.println();
-
         personPerson.printPersonDetails();
         System.out.println();
-
         personPerson.getIsEmailRestricted();
 
+        Map<String, Integer> map = new HashMap<String, Integer>();
+        map.put("romana", 6);
+        map.put("matematica", 8);
+        map.put("desen", 10);
+        map.put("geografie", 9);
+        map.put("muzica", 7);
+        map.put("civica", 10);
+
+        Student student = new Student("Email", "Popescu", "Male", 21, "email.pop@yahoo.com", false, true, "12", 98, map);
+
+        System.out.println("Check discipline marks: ");
+        student.addDisciplineMark("biologie", 8);
+        student.addDisciplineMark("informatica", 9);
+        map.remove("civica");
+        map.replace("geografie", 3);
+
+        int note1 = map.get("geografie");
+
+        try {
+            int note2 = map.get("civica");
+
+        } catch (NullPointerException e) {
+            System.out.println("it's just an exception named NullPointerException ");
+        }
+
+        int note3 = map.get("desen");
+        System.out.println("Geografie = " + note1);
```

Comment:
short remark: you could also retrieve the note and print it directly, without using the String helper variable (to save memory space), like this: `System.out.println("Geografie = " + map.get("geografie"));`

Reply:
Changed to remarked format in order to save unnecessary variables.